lib/elixir/lib/string.ex

```diff
@@ -68,8 +68,9 @@ defmodule String do
 
   ## Code points and grapheme cluster
 
-  The functions in this module act according to the Unicode
-  Standard, version 13.0.0.
+  The functions in this module act according to
+  **The Unicode Standard, Version 13.0.0**.
+  <http://www.unicode.org/versions/Unicode13.0.0/>
```

Comment:
I believe the context of that note is in relation to academic or official references, but it doesn't hurt to be clearer.

```suggestion
  [The Unicode Standard, Version 13.0.0](http://www.unicode.org/versions/Unicode13.0.0/).
```